packages/amplify-cli/src/execution-manager.ts

```diff
@@ -39,6 +39,33 @@ export function isContainersEnabled(context) {
 }
 
 async function selectPluginForExecution(context: Context, pluginCandidates: PluginInfo[]): Promise<PluginInfo> {
+  const pluginCandidatesCategorySet = new Set<string>();
+  const overidedcategories = ['auth'];
```

Comment:
overrideCategories - var spell check

Reply:
Can we read this from here instead - https://github.com/aws-amplify/amplify-cli/pull/8228/files#diff-8d86b03f9e5e5dc9ddd1d9ebe2ffcda3ac67c29234888d7c50317afcb57155b4R29 ??

Reply:
done